### PR TITLE
docs(grid) Documented expose aggregates for all fields

### PR DESCRIPTION
**Pull request by NansiYancheva**
Related to https://github.com/telerik/blazor/issues/8848

### CODE REVIEW

File: components/grid/grouping/aggregates.md [hunk 1]
```diff
@@ -156,7 +169,7 @@ Enable and use aggregates. To see the full effect, group by a column - "Team" an
 
 * You should define only aggregates that you will use to avoid unnecessary calculations that may be noticeable on large data sets.
 
-* If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will be thrown.
+* If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will be thrown or you will not get the result from the aggregation.
```
Comment (dimodi) [inline components/grid/grouping/aggregates.md:172]:
```suggestion
* If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will occur or you will get a `null` value.
```

File: components/grid/grouping/aggregates.md [hunk 2]
```diff
@@ -41,55 +41,62 @@ You can use aggregates in the following templates:
 * [`GroupHeaderTemplate`]({%slug grid-templates-group-header%}) of a `GridColumn` - a header in the respective column that renders when the grid is grouped by that column. The `Value` field in the context carries the current group value.
 * [`FooterTemplate`]({%slug grid-templates-column-footer%}) of a `GridColumn` - a grand total row of footers for the entire grid.
 
+## Access The Aggregate Values
+
+You can access the aggregate values through the template `context`.
+
+In all templates, you can access the aggregate values available for the column, in which the template is nested.
+
+In both the `GroupHeaderTemplate` and the `GroupFooterTemplate`, the `context` has an `AggregateResults` property of a type Dictionary. This allows you to access the aggregates available for each column in the Grid.
```
Comment (dimodi) [inline components/grid/grouping/aggregates.md:50]:
```suggestion
You can access the aggregate values through the template `context`:

* All templates expose the aggregate values for the current column.
* The `context` of the `GroupHeaderTemplate` and the `GroupFooterTemplate` has an `AggregateResults` property of a type `Dictionary<string, object>`. This dictionary allows you to access the aggregates for the other columns in the Grid.
```

File: components/grid/grouping/aggregates.md [hunk 3]
```diff
@@ -111,17 +118,28 @@ Enable and use aggregates. To see the full effect, group by a column - "Team" an
                     }
                 }
             </GroupHeaderTemplate>
+            <GroupFooterTemplate>
+                @*access the aggregates of the column in which the template is*@
+                All active projects: @context.Sum
+                <br />
+                @*access the aggregates of all columns*@
```
Comment (dimodi) [inline components/grid/grouping/aggregates.md:125]:
```suggestion
                @*access the aggregates of the other columns*@
```

File: components/grid/grouping/aggregates.md [hunk 4]
```diff
@@ -111,17 +118,28 @@ Enable and use aggregates. To see the full effect, group by a column - "Team" an
                     }
                 }
             </GroupHeaderTemplate>
+            <GroupFooterTemplate>
+                @*access the aggregates of the column in which the template is*@
```
Comment (dimodi) [inline components/grid/grouping/aggregates.md:122]:
```suggestion
                @*access the aggregates of the ActiveProjects column*@
```

File: components/grid/grouping/aggregates.md [hunk 5]
```diff
@@ -111,17 +118,28 @@ Enable and use aggregates. To see the full effect, group by a column - "Team" an
...
+                All active projects: @context.Sum
+                <br />
+                @*access the aggregates of all columns*@
+                Total employees and teams: @(context.AggregateResults[nameof(Employee.Team)].Count + context.AggregateResults[nameof(Employee.Name)].Count)
+                <br />
```
Comment (dimodi) [inline components/grid/grouping/aggregates.md:127]:
```suggestion
```

File: components/grid/grouping/aggregates.md [hunk 6]
```diff
@@ -156,7 +174,7 @@ Enable and use aggregates. To see the full effect, group by a column - "Team" an
 
 * You should define only aggregates that you will use to avoid unnecessary calculations that may be noticeable on large data sets.
 
-* If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will be thrown.
+* If you try to use an aggregate that is not defined, or an aggregate over an unsupported field type, a runtime error will occur or you will get a `null` value.
```
Comment (dimodi) [inline components/grid/grouping/aggregates.md:177]:
```suggestion
* If you try to use an aggregate that is not defined, you will get a `null` value.
* If you try to use an aggregate that is not compatible with `Field` type, a runtime error will occur.
```

**Review by dimodi — APPROVED** [review]

**Review by ikoevska — COMMENTED** [review]
Reviewed.

**Pull request merged**